### ISSUE TITLE
Unable to Expand Database Connection To See Tables, Etc. 

### ISSUE THREAD

**Issue by jbunselmeyer** [mtxr/vscode-sqltools#266]
I am a new user to SQLTools. I was able to install the extension and create several connections to SQL Server database servers. I appear to be able to connect to those connections normally (icon is yellow) but when I attempt to expand a connection that is currently connected no additional items (tables, etc.) are displayed. 

**To Reproduce**
1. Click on Connect (plug) icon for a Connection that was just created. 
2. Database icon turns yellow. 
3. Click arrow/triangle to left of database icon and no tables, etc. are displayed. 

**Expected behavior**
After clicking arrow to left of database icon tables, etc. should be displayed. 

**Screenshots**
![image](https://user-images.githubusercontent.com/12647017/59127988-97375e80-892e-11e9-81d0-1c7ab7d95db9.png)
![image](https://user-images.githubusercontent.com/12647017/59128122-bfbf5880-892e-11e9-9bda-6c04162ba126.png)
![image](https://user-images.githubusercontent.com/12647017/59127797-16786280-892e-11e9-9788-78bd520e0ba5.png)

**Desktop (please complete the following information):**
 - SQLTools Version: 0.19.5 
 - VSCode Version: 1.35.0 (user)
 - Dialect Used: MSSQL
 - OS: Windows 10 1703

**Comment by jbunselmeyer** [comment]
May be related to #262 as the end results were similar (showed connected, clicking arrow/triangle to expand shows nothing).

**Comment by mtxr** [comment]
Hi, 

sorry for the late reply, I had to deal with some stuff here in the past week, I'll take a look at this within this week.

Thanks!

**Comment by jbunselmeyer** [comment]
A quick update/clarification. I can run queries successfully so this issue may differ from #262 in that respect.

**Comment by mtxr** [comment]
@jbunselmeyer can you share the logs of SQLTools output channels?

Is you DB huge?

**Comment by jbunselmeyer** [comment]
Database is ~250GB. 

DEBUG: Configs reloaded!
INFO: Telemetry enabled!
DEBUG: Event: sessionStarted:extension 
DEBUG: Activation time: 12.683082ms
ERROR: (node:15968) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
WARN: Git installation not found.
INFO: Executing SQLTools.selectConnection

I connect successfully but nothing displays when I click the triangle/arrow to expand. Nothing additional is logged in Output with SQLTools selected.

**Comment by mtxr** [comment]
Closing in favor of #322.

**Comment by mtxr** [comment]
@jbunselmeyer can you confirm this issue still exists?

**Comment by jbunselmeyer** [comment]
Yes, it still exists. In fact, in v0.21.0 I've noticed the connections section looks a bit different and now I don't even have the arrow to the left of the database icon that I would assume is suppose to be an "expand" function that would show the schema, etc.

![image](https://user-images.githubusercontent.com/12647017/65817378-c61e5c80-e1cb-11e9-9767-743b78d35988.png)

**Comment by mtxr** [comment]
Would you mind sending the logs? We should see something logged on output channels. I can't reproduce it here

**Comment by jbunselmeyer** [comment]
Not much to provide. All it logs when I open VS Code and connect to a database is this...

DEBUG: Settings updated!
INFO: Telemetry enabled!
DEBUG: Event: sessionStarted:extension 
DEBUG: Activation time: 13.985231ms
INFO: Executing SQLTools.getConnectionStatus
DEBUG: Connection explorer updated.
INFO: Executing SQLTools.selectConnection
DEBUG: Connection explorer updated. Updated Help Desk
INFO: Executing SQLTools.getConnectionStatus
DEBUG: Connection explorer updated.
DEBUG: Connection explorer updated. Updated Help Desk
INFO: Executing SQLTools.getConnectionStatus

Since I now don't even see the arrow to the left of the database icon that I would assume is suppose to be an "expand" function that would show the schema, etc. there isn't much else I can do besides connect as I don't see a way to "expand" once I've connected to the database.
